### DISCUSSION TITLE
How to set multiple cookie for bing?

### DISCUSSION THREAD

**Discussion by Ayndpa** [waylaidwanderer/node-chatgpt-api#318] [Help]
I want to set more than one cookie for bing because I'm worried that one will reach the maximum daily limit.What should I do?

**Comment by WANHO-O** [comment]
`const sydneyAIClient = [
    new BingAIClient({
        ...bingOptions_1,
        cache: bingCacheOptions,
    }),
    new BingAIClient({
        ...bingOptions_2,
        cache: bingCacheOptions,
    }),
    new BingAIClient({
        ...bingOptions_3,
        cache: bingCacheOptions,
    }),
    new BingAIClient({
        ...bingOptions_4,
        cache: bingCacheOptions,
    }),
];`
My method is to create multiple clients, each with a different configuration

**Reply by Ayndpa** [reply]
I modified some places in the source code and realized a simple dynamic switching of cookie.
